src/Http/Headers/src/SameSiteMode.cs

```diff
@@ -3,10 +3,16 @@
 
 namespace Microsoft.Net.Http.Headers
 {
-    // RFC Draft: https://tools.ietf.org/html/draft-ietf-httpbis-cookie-same-site-00
+    /// <summary>
+    /// Determines whether to send a cookies on "same-site" or "cross-site" requests
+    /// RFC Draft: https://tools.ietf.org/html/draft-ietf-httpbis-cookie-same-site-00
+    /// </summary>
+    /// <remarks>
+    /// This mirrors Microsoft.AspNetCore.Http.SameSiteMode
```

Comment:
I think the remarks can be omitted because they're really just "for us" - so maybe leave those as a `//` comment?